### ISSUE TITLE
Doc missing for `neil dep versions`

### ISSUE THREAD

**Issue by teodorlu** [babashka/neil#108]
Entrypoint for `neil dep versions` appears to be missing from the global helptext. But the command works fine :)

```
# On master, as of 2022-08-16
$ (cd ~/dev/teodorlu/neil && git rev-parse HEAD)
e21bc88cdb0cfcb9e9d3bb4f5d8640885735ca9f
$ neil
Usage: neil <subcommand> <options>

Most subcommands support the options:
  --alias      Override alias name.
  --deps-file  Override deps.edn file name.

Subcommands:

add
  dep    Alias for `neil dep add`.
  test   adds cognitect test runner to :test alias.
  build  adds tools.build build.clj file and :build alias.
  kaocha adds kaocha test runner to :koacha alias.
  nrepl  adds nrepl server to :nrepl alias.

dep
  add: Adds --lib, a fully qualified symbol, to deps.edn :deps.
    Run `neil dep add --help` to see all options.

  search: Search Clojars for a string in any attribute of an artifact
    Run `neil dep search --help` to see all options.

new:
  Create a project using deps-new
    Run neil new --help to see all options.

test:
  Run tests. Assumes `neil add test`. Run `neil test --help` to see all options.

license
  list   Lists commonly-used licenses available to be added to project. Takes an optional search string to filter results.
  search Alias for `list`
  add    Writes license text to a file
    Options:
    --license The key of the license to use (e.g. epl-1.0, mit, unlicense). --license option name may be elided when license key is provided as first argument.
    --file    The file to write. Defaults to 'LICENSE'.
# But `dep versions` still works fine
$ neil dep versions hiccup/hiccup
:lib hiccup/hiccup :version 2.0.0-alpha2
:lib hiccup/hiccup :version 2.0.0-alpha1
:lib hiccup/hiccup :version 1.0.5
:lib hiccup/hiccup :version 1.0.4
:lib hiccup/hiccup :version 1.0.3
:lib hiccup/hiccup :version 1.0.2
:lib hiccup/hiccup :version 1.0.1
:lib hiccup/hiccup :version 1.0.0
:lib hiccup/hiccup :version 1.0.0-RC3
:lib hiccup/hiccup :version 1.0.0-RC2
```

**Comment by Cyrik** [comment]
It seems to only work for stuff in clojars, e.g. org.clojure/core.async doesn't have versions.

**Comment by borkdude** [comment]
@Cyrik Please post a new issue about this

**Comment by borkdude** [comment]
@teodorlu Is this issue still relevant?

**Comment by teodorlu** [comment]
I'd say yes. I can't see anywhere from the helptext to find the `neil dep versions` command.

As of `neil 0.1.48`,

1. I don't see any mention of `neil dep versions` in the global helptext
2. `neil dep` prints the global helptext
3. `neil dep versions` without any arguments just crashes

**Comment by teodorlu** [comment]
Fixed as of #155.
